var/spack/repos/builtin/packages/nccl/package.py

```diff
@@ -58,7 +58,20 @@ class Nccl(MakefilePackage, CudaPackage):
 
     variant("cuda", default=True, description="Build with CUDA")
 
-    depends_on("rdma-core")
+    # Make verbs default but package like aws-ofi-nccl can be used instead which loads
+    # another library to use instead at runtime.
+    variant(
+        "fabrics",
+        values=disjoint_sets(
+            ("auto"),
+            (
+                "verbs",
+            ),  # supported transports
+        ).with_default("verbs").with_non_feature_values("auto"),
+        description="List of fabrics that are enabled; " "'auto' lets nccl determine at runtime",
```

Comment:
```suggestion
        description="List of fabrics that are enabled; 'auto' lets nccl determine at runtime",
```

Reply:
I think this is due to copying from OpenMPI code e.g. https://github.com/spack/spack/blob/68570b75874c1312d463d1d43907a2b95db8c76f/var/spack/repos/builtin/packages/openmpi/package.py#L474